src/tests/component/Auth.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Auth from '../../components/Auth/Auth';
+
+describe('<Auth />', () => {
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
```

Comment:
Declare the `wrapper` outside or in a before block so that you do not have to declare it in each test suit.